cmake/leveldb.cmake

```diff
@@ -99,3 +99,7 @@ target_link_libraries(leveldb PRIVATE
   nowarn_leveldb_interface
   crc32c
 )
+
+set_target_properties(leveldb PROPERTIES
+  EXPORT_COMPILE_COMMANDS OFF
```

Comment:
Why do we need to set this for some of our targets?

Reply:
By default, when `EXPORT_COMPILE_COMMANDS` is `ON`, CMake generates a `compile_commands.json`, which is used in the "tidy" CI task. On the master branch, `compile_commands.json` is generated by the `bear` tool.

We do not want to analyze code from subtrees, so disabling `EXPORT_COMPILE_COMMANDS` in there.

Reply:
Isn't it supposed to be done in upstream repositories?